client/src/app/pages/applications/analysis-wizard/set-options.tsx

```diff
@@ -77,6 +77,17 @@ export const SetOptions: React.FC = () => {
     return 0;
   });
 
+  const labelsSeen = new Set();
+  const uniqueTargetsAndLabels = defaultTargetsAndTargetsLabels.filter(
+    (target) => {
+      if (labelsSeen.has(target.label)) {
+        return false;
+      }
+      labelsSeen.add(target.label);
+      return true;
+    }
+  );
+
```

Comment:
Looks like this doesn't fix the dupe issue since the objects from the two arrays are constructed separately. Set isn't working how we'd expect here. Need to do the concat before the set I think.